logger.py

```diff
@@ -9,7 +9,8 @@
 
 logging.basicConfig(
     level=logging.DEBUG,
-    format='%(asctime)s.%(msecs)03d | \t %(levelname)s:\t %(message)s',
+    # format='%(asctime)s.%(msecs)03d | \t %(levelname)s:\t %(message)s',
+    format='%(asctime)s - %(pathname)s[line:%(lineno)d] - %(funcName)s - %(levelname)s: %(message)s',
```

Comment:
I don't this it's necessary to have all trace info in the log. They could be located easily and the trace will mess the log output.